src/components/CardHeader.tsx

```diff
@@ -0,0 +1,45 @@
+import { nullable, Link, Text } from '@taskany/bricks';
+import { gray8 } from '@taskany/colors';
+import styled from 'styled-components';
+
+interface CardHeaderProps extends React.HTMLAttributes<HTMLDivElement> {
+    name?: string | null;
+    timeAgo: React.ReactNode;
+    href?: string;
+}
+
+const StyledCardHeader = styled.div`
+    display: inline-flex;
+`;
+
+const StyledSeparator = styled.span`
+    margin-left: 0.25rem;
+`;
+
+const Separator = () => <StyledSeparator>—</StyledSeparator>;
```

Comment:
U can use CSS content property and not create this component) but as u wish

Reply:
No, cause on hover pseudo element will apply text styles from linked element